app/models/album.js

```diff
@@ -54,5 +50,23 @@ module.exports = (sequelize, DataTypes) => {
     }
   };
 
+  Album.findAlbumsByUser = async req => {
```

Comment:
this work but i think that something like: 
```
Album.findAlbumsByUser = req => {
  if (!req.decoded.admin && Number(req.params.id) !== req.decoded.id) {
    logger.error(`You must have admin permissions to get the albums of another user`);
    throw unauthorizedError(‘You must have admin permissions to get the albums of another user’);
  }
  logger.info(`Searching the albums of user ${req.params.id}...`);

  return Album.findAll({ where: { userId: req.params.id } })
  .then(albums => albums)
  .catch(err => {
    logger.error(`A database error has occurred during the search of albums`);
    throw databaseError(err);
  });
};
```
its better and more readable.

and in line 44 change `return` for `throw`